spot-ml/ml_test.sh

```diff
@@ -22,28 +22,10 @@ HPATH=${HUSER}/${DSOURCE}/test/scored_results
 FEEDBACK_PATH=${LPATH}/${DSOURCE}_scores.csv
 DUPFACTOR=1000
 
-HDFS_WORDCOUNTS=${HPATH}/word_counts
-
-# paths for intermediate files
-HDFS_DOCRESULTS=${HPATH}/doc_results.csv
-LOCAL_DOCRESULTS=${LPATH}/doc_results.csv
-
-HDFS_WORDRESULTS=${HPATH}/word_results.csv
-LOCAL_WORDRESULTS=${LPATH}/word_results.csv
-
 HDFS_SCORED_CONNECTS=${HPATH}/scores
-HDFS_MODEL=${HPATH}/model
-
-LDA_OUTPUT_DIR=test/${DSOURCE}
 
 TOPIC_COUNT=20
```

Comment:
I just noticed this.... but TOPIC_COUNT should really go into the spot.conf files

we should have all the user-customizable state in one place, not scattered around different files